tests/coverage/mcdc/if.cov-map

```diff
@@ -1,4 +1,4 @@
-Function name: mcdc_if::mcdc_check_a
```

Comment:
Renaming the test files is fine, but I don't think we should rename the functions in the files.

It makes the history harder to follow, and there isn't any corresponding benefit.